spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -780,13 +781,39 @@ def download_key(id: nil)
     def create_key!(name: nil, service_configs: nil)
       fetch_csrf_token_for_keys
 
+      service_configs_requests = (service_configs || {}).map do |service_id, configs|
+        if service_id == Spaceship::Portal::Key::MUSIC_KIT_ID
+          {
+            serviceId: service_id,
+            isNew: true,
+            identifiers: configs[:identifiers] || {},
+            }
```

Comment:
Could you double check this area? The indents on the `{}` look off.

Reply:
Fixed